### ISSUE TITLE
Converted Excel has blank line

### ISSUE THREAD

**Issue by pslobster** [toolbox-labo/md-design-doc#20]
In converted excel file, create a blank line at first line of heading and description area.

md
```
# Sheet Name
## Test Variation 1
### Test Variation 1-1
#### Test Variation 1-1-1
* Test Description
  more lines...
  more lines...
### Test Variation 1-2
#### Test Variation 1-2-1
* Test Description
  more lines...
  more lines...
#### Test Variation 1-2-2
##### Test Variation 1-2-2-1
* Test Description
  more lines...
  more lines...
```

Excel

<img width="682" alt="スクリーンショット 2021-01-28 3 32 30" src="https://user-images.githubusercontent.com/50319283/106037378-f0fb2680-6119-11eb-8952-a27780719536.png">

[test.xlsx](https://github.com/toolbox-labo/md-design-doc/files/5882322/test.xlsx)

**Comment by ymtdzzz** [comment]
This seems to be the cause:

https://github.com/toolbox-labo/md-design-doc/blob/49514077deda6d600a40e3cedc7a7f96b5b54e1d/md_designer/src/data.rs#L220-L225

**Comment by vivaquo** [comment]
May be fixed this way...

```
    fn concat(target: &Option<&String>, input: &str) -> String {
        if let Some(str) = target {
            if str.len() > 0 {
                format!("{}\n{}", str, input)
            } else {
                format!("{}", input)
            }
        } else {
            input.to_string()
        }
    }
```

**Comment by ymtdzzz** [comment]
This issue has been solved in #37.
